notification-service/.flattened-pom.xml

```diff
@@ -142,6 +195,24 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.springdoc</groupId>
+        <artifactId>springdoc-openapi-maven-plugin</artifactId>
+        <version>1.4</version>
+        <executions>
+          <execution>
+            <id>integration-test</id>
+            <phase>package</phase>
+            <goals>
+              <goal>generate</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <outputFileName>api-docs.json</outputFileName>
```

Comment:
As per feedback received for `service-sdk`, filename should be `openapi.json`. If I'm not wrong, It's the default value (but please double check :) )


>Document Structure
> An OpenAPI document MAY be made up of a single document or be divided into multiple, connected parts at the discretion of the author. In the latter case, Reference Objects and Schema Object $ref keywords are used.
It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.
https://swagger.io/specification/

Reply:
Thanks for the tip, changed it
I also removed the `openapi` directory, file is created under `generated` directory